src/applications/edu-benefits/components/GetFormHelp.jsx

```diff
@@ -5,27 +5,16 @@ function GetFormHelp() {
   return (
     <div>
       <p className="help-talk">
-        For help filling out this form,
-        <br />
-        ask the Education Call Center:
-      </p>
-      <p className="help-phone-number">
-        <a className="help-phone-number-link" href="tel:1-888-442-4551">
-          888-442-4551 (888-GI-BILL-1)
+        Call us at{' '}
+        <a href="tel:800-827-1000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">
+          800-827-1000
         </a>
-        <br />
-        Monday &#8211; Friday, 8:00 a.m. &#8211; 7:00 p.m. ET
-        <br />
-        <a
-          className="help-phone-number-link"
-          href="https://gibill.custhelp.com/app/utils/login_form/redirect/ask"
-        >
-          Submit a question to Education Service
+        . We’re here Monday through Friday, 8:00 am to 9:00 pm ET. If you have
```

Comment:
Please use "a.m." and "p.m." here ([ref](https://design.va.gov/content-style-guide/dates-and-numbers#times-and-time-zones)). Also, screenreaders would say "am" and "pm" as words and not separate letters.

Reply:
Just refactored to include this.